.github/workflows/create-patch-pr.yml

```diff
@@ -0,0 +1,58 @@
+name: 'Create Patch PR'
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit:
+        description: 'The commit SHA to cherry-pick for the patch.'
+        required: true
+        type: 'string'
+      channel:
+        description: 'The release channel to patch.'
+        required: true
+        type: 'choice'
+        options:
+          - 'stable'
+          - 'preview'
+      dry_run:
+        description: 'Whether to run in dry-run mode.'
+        required: false
+        type: 'boolean'
```

Comment:
Super nit (for consistency): consider updating this from `'boolean'` to `boolean`